src/bindings/text-writer.cc

```diff
@@ -34,11 +34,17 @@ void TextWriter::construct(const Nan::FunctionCallbackInfo<Value> &info) {
 void TextWriter::write(const Nan::FunctionCallbackInfo<Value> &info) {
   auto writer = Nan::ObjectWrap::Unwrap<TextWriter>(info.This());
 
-  if (info[0]->IsString()) {
-    Local<String> js_chunk = info[0]->ToString();
+  Local<String> js_chunk;
+  if (Nan::To<String>(info[0]).ToLocal(&js_chunk)) {
     size_t size = writer->content.size();
     writer->content.resize(size + js_chunk->Length());
     js_chunk->Write(
+
+// Nan doesn't wrap this functionality
+#if NODE_MAJOR_VERSION >= 12
+      Isolate::GetCurrent(),
+#endif
```

Comment:
Here's another case where we really want to use `String::Write` for performance, because it allows us to reuse the `std::string` that we've already allocated.